pallets/phala/src/fat.rs

```diff
@@ -377,26 +386,6 @@ pub mod pallet {
 				_ => return Err(Error::<T>::InvalidSender.into()),
 			};
 			match message.payload {
-				WorkerContractReport::CodeUploaded {
```

Comment:
@shelvenzhou FYI. This event will be deleted. In my understanding it's not used in the e2e tests of the phala-blockchain repo or the oracle-workshop repo. But I'm not sure if it can affect the behavior of Contracts UI (hopefully no). We should let all the downstream to notice it.